package.json

```diff
@@ -319,7 +319,6 @@
     "@types/opn": "^5.1.0",
     "@types/podium": "^1.0.0",
     "@types/prop-types": "^15.5.3",
```

Comment:
Moved this over to the x-pack package.json where it should have been (my own fault)